src/Microsoft.Management.Configuration.OutOfProc/Factory.h

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#pragma once
+#include <hstring.h>
+#include <inspectable.h>
+#include <winrt/Windows.Foundation.h>
+#include <atomic>
+
+namespace Microsoft::Management::Configuration::OutOfProc
+{
+    struct Factory : winrt::implements<Factory, winrt::Windows::Foundation::IActivationFactory, IClassFactory>
+    {
+        Factory();
+        ~Factory();
+
+        // Determines if the reference count is 0.

```

Comment:
is not 0?

Reply:
It is a boolean, you get to know both whether it is 0 AND whether it is not 0.